18 ReactHotLoader/webpack.config.js

```diff
@@ -14,35 +15,32 @@ module.exports = {
       extensions: ['', '.js', '.ts', '.tsx']
   },
 
+  // Toake into account: remove hot loading entry points in production
   entry: [
+    'webpack-dev-server/client?http://localhost:8080',
+    'webpack/hot/only-dev-server',
     './index.tsx',
     './css/site.css',
     '../node_modules/toastr/build/toastr.css',
     '../node_modules/bootstrap/dist/css/bootstrap.css'
   ],
 
   output: {
+    publicPath: 'http://localhost:8080/',
     path: path.join(basePath, "dist"),
     filename: 'bundle.js'
   },
 
-  //https://webpack.github.io/docs/webpack-dev-server.html#webpack-dev-server-cli
-  devServer: {
```

Comment:
You can still using this configuration instead of create devServer.js and more dependencies like express.

Configuration:

```
devServer: {
    contentBase: './dist', //Content base
    inline: true, //Enable watch and live reload
    host: 'localhost',
    port: 8080,
    hot: true
  },
```